src/core_plugins/elasticsearch/lib/map_uri.js

```diff
@@ -19,29 +17,29 @@ export default function mapUri(server, prefix) {
       port: esUrlPort,
       pathname: esUrlBasePath,
       query: esUrlQuery
-    } = parseUrl(config.get('elasticsearch.url'), true);
+    } = parseUrl(cluster.config('url'), true);
 
     // copy most url components directly from the elasticsearch.url
     const mappedUrlComponents = {
       protocol: esUrlProtocol,
-      slashes: esUrlHasSlashes,
-      auth: esUrlAuth,
+      slashes: esUrlHasSlashes, // QUESTION: would it every be anything else?
```

Comment:
What are these questions about?  Are they still unanswered?  We should figure any outstanding questions like this out before merging.